Added the section of Offliner packager (#412)
opengisch/QField-docs: documentation/get-started/tutorials/get-started-qfc.en.md

You will receive notifications for events in which you are not the actor. These notifications are specifically for events that are initiated by other members of your organization or collaborators on your projects.

## Enhance your project with the "Optimized Packager"

> For new projects, it is recommended to use the new "Optimized Packager" over the deprecated "QGIS Core Offline Editing". Set it under "Packaging Offliner" in the "Settings" tab of your project. The "Optimized Packager" supports consolidating filtered layers of same datasource into a single oflline layer, respecting distinct symbology. This also uses less storage. Here's an example to illustrate this feature:

Reviewer: ```suggestion
We recommend to use the new "Optimized Packager" over the deprecated "QGIS Core Offline Editing" for all your projects. Set the packager under "Packaging Offliner" in the "Settings" tab of your project.

The "Optimized Packager" supports consolidating filtered layers of same datasource into a single oflline layer, respecting distinct symbology but also using less storage. Here's an example to illustrate this feature:
```

You will receive notifications for events in which you are not the actor. These notifications are specifically for events that are initiated by other members of your organization or collaborators on your projects.

## Enhanced your projects with the Offliner packaging.

> The new Packaging Offliner (in "Settings" of your projects) introduces advanced capabilities, allowing the support of multiple layers using the same data source but with distinct filters. This enhancement offers greater flexibility in offline datasources management. Here's an example to illustrate this feature:

Reviewer 2: ```suggestion
For new projects, it is recommended to use the new "Optimized Packager" over the deprecated "QGIS Core Offline Editing". Set it under "Packaging Offliner" in the "Settings" tab of your project. The "Optimized Packager" supports consolidating filtered layers of same datasource into a single oflline layer, respecting distinct symbology. This also uses less storage. Here's an example to illustrate this feature:
```

### Example Configuration:

- **Layer 1.1:**
  - Data Source: `layers.gpkg`
  - Table: `layer1`
  - Filter: `id % 2 = 1`

- **Layer 1.2:**
  - Data Source: `layers.gpkg`
  - Table: `layer1`
  - Filter: `id % 2 = 0`

### Result:

For the new offliner:
- A single layer is generated in the offline geopackage, combining data from `layer1` with the specified filters.

For the old (QGIS) offliner:
- Two separate layers are created, each representing the filtered datasets:
  - Layer 1: Filtered with `id % 2 = 1`
  - Layer 2: Filtered with `id % 2 = 0`

### Advantages:

1. **Efficiency:**

>    - The new offliner configuration optimizes storage by consolidating filtered layers using the same datasource, enhancing efficiency in data management, and for apply distinct  symbology to the same layer.

Reviewer 2: ```suggestion
   - The new offliner uses less storage by consolidating filtered layers into a single datasource respecting distinct symbology.
```

> 
> 2. **Compatibility:**
>    - The Packaging Offliner is designed to package data for offline use with QField. For new projects, it is recommended to use the "Optimized Packager" over the deprecated "QGIS Core Offline Editing." The Optimized Packager offers improved performance and enhanced features for offline data management.

Reviewer 2: ```suggestion

```
this i would discard

> 
> ### Advantages:
> 
> 1. **Efficiency:**
>    - The new offliner configuration optimizes storage by consolidating filtered layers using the same datasource, enhancing efficiency in data management, and for apply distinct  symbology to the same layer.
> 
> 2. **Compatibility:**
>    - The Packaging Offliner is designed to package data for offline use with QField. For new projects, it is recommended to use the "Optimized Packager" over the deprecated "QGIS Core Offline Editing." The Optimized Packager offers improved performance and enhanced features for offline data management.

Reviewer 2: ```suggestion

```
I would drop this
